dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
```

Comment:
It's against the [Guidelines][0] to gone pass 80-col limit.

```diff
-DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+DESCRIPTION="A library to support the benchmarking of functions, similar to \
+unit-tests."
```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips